custom_components/tahoma/__init__.py

```diff
@@ -208,9 +209,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     supported_platforms = set(platforms.keys())
 
     # Sensor and Binary Sensor will be added dynamically, based on the device states
-    supported_platforms.add(BINARY_SENSOR)
-    supported_platforms.add(SENSOR)
-
+    # Required to load the low speed switch
```

Comment:
```suggestion
    # Switch will be added dynamically, based on device features (e.g. low speed cover switch)
```

Should we change it to something more descriptive?